ApiClient.js

```diff
@@ -1,7 +1,28 @@
 var config = require('./config');
+import {AsyncStorage} from 'react-native';
 
+var token;
 export default class ApiClient {
-	static get(endpoint, headers) {
+	static async getAuthToken(){
+		if(token != undefined) return token;
+		token = await AsyncStorage.getItem('@Skybunk:token');
+		return token;
+	}
+
+	static async setAuthToken(_token){
+		token = _token;
+		await AsyncStorage.setItem('@Skybunk:token', token);
+	}
+
+	static async clearAuthToken(){
+		await AsyncStorage.removeItem('@Skybunk:token');
+		token = undefined;
+	}
+
+	static async get(endpoint, options={}) {
+		if(options.authorized) var headers = {'Authorization': 'Bearer ' + await this.getAuthToken(), ...options.headers}
+		else var headers = options.headers
```

Comment:
This just looks super awkward, what would you think about 

```
let headers = {...options.headers}
if(options.authorized) {
    headers = {
        ...headers,
        'Authorization': `Bearer ${await this.getAuthToken()}`
    }
}
```

Or even moving the logic into a method so we can do something like
```
const headers = formatHeaders(options)
```
or something since it's used in multiple places.